lawlibrary/templates/lawlibrary/home.html

```diff
@@ -38,44 +38,6 @@ <h2 class="mb-4 heading-underlined">{% trans 'Recent Legislation' %}</h2>
     {% trans 'View more legislation' as text %}
     {% include 'peachjam/_recent_document_list.html' with documents=recent_legislation view_more_text=text view_more_url_name='legislation_list' %}
   </section>
-  <section class="pb-3 pt-3">
-    <h2 class="mb-4 heading-underlined">{% trans 'Collections' %}</h2>
-    {% include 'peachjam/_taxonomy_list.html' %}
-  </section>
-  <section class="py-5">
```

Comment:
Pocketlaw and constitution compass only apply to lawlibrary, and need to stay here.

Collections can go to liiweb.